packages/mpl-scatter-density.yml

```
@@ -0,0 +1,3 @@
+repo: astrofrog/mpl-scatter-density
+section: miscellaneous
```

I think this and datashader are plot types ?

---

Yes and no, but sure!

```suggestion
section: plot types
```